README.md

```diff
@@ -5,3 +5,158 @@
 - Web and Network  
 - Software Development
 - Web Development
+
+### BASIC COMPUTER TERMS
+
+#### Computer
+A computer is a device that executes a program or programs. 
+
+#### Hardware:
+Hardware consists of the physical components of your computer. If you can kick it, punch it, or pick it up to throw it, it’s hardware. For e.g. Mouse, Keyboard, Monitor are hardware
+
+#### Software	
+A software is a program application; which provides the computer with step-by-step instructions to perform a specific task. A computer cannot function without software.
+
+#### Application Program
+A Program that has been created to perform a specific task that is useful to the user. Examples include Word Processors (Word), Spreadsheets (Excel), Browsers (Netscape) and Drawing Packages (Paint).
+
+#### CPU 
+The central processing unit or CPU, also known as the microprocessor or processor. The CPU performs basic arithmetic, logic, controlling, and input/output (I/O) operations specified by the instructions
+
+#### Operating System  
+The operating system (OS) of a computer is the software that communicates with the computer hardware. No software can run on the computer without the operating system.
+
+#### File 
+A file is a block of information that includes word documents, pictures, music, etc. Each file is differentiated with the help of a unique file name. 
+
+#### Folder 
+A folder is a system that helps in organizing files, programs and projects on the computer. It is similar to organizing files on a shelf in your cabinet. 
+
+#### Memory	
+Chips that hold information that the computer needs to use. These chips are connected directly to the Microprocessor. There are two types of Memory Chip: Random Access Memory (RAM) and Read Only Media (ROM)
+
+##### RAM
+> RAM (random access memory) is the memory of a computer which works actively with the hard drive. It pulls the data when it’s in use then returns it back to the hard drive once you’re done with it.
+
+##### ROM
+> ROM is a storage medium that is used with computers and other electronic devices. As the name indicates, data stored in ROM may only be read.
+
+#### Modem 
+A modem is something that is connected to a computer to connect it to the internet or other networks. This is typically in the form of a small box that you get from your provider.
+
+#### Hard Disk	
+The Hard Disk is where the data is stored within the computer.
```

Comment:
All your files (images, music and softwares) and also your OS are stored in hard drive